include/amber/amber_vulkan.h

```diff
@@ -15,16 +15,43 @@
 #ifndef AMBER_AMBER_VULKAN_H_
 #define AMBER_AMBER_VULKAN_H_
 
+#include <string>
+#include <vector>
+
+#include "amber/amber.h"
 #include "vulkan/vulkan.h"
 
 namespace amber {
 
 /// Configuration for the Vulkan Engine.
 struct VulkanEngineConfig : public EngineConfig {
-  /// The VkDevice to use for the tests.
-  VkDevice device;
+  /// The VkPhysicalDevice to be used for the tests.
+  VkPhysicalDevice physical_device = VK_NULL_HANDLE;
```

Comment:
I thought if we passed in the extensions, queue and features we wouldn't need physical_device anymore?